src/vs/editor/standalone/browser/standaloneEditor.ts

```diff
@@ -456,6 +459,36 @@ export function registerLinkOpener(opener: ILinkOpener): IDisposable {
 	});
 }
 
+export interface ICodeEditorOpener {
+	openCodeEditor(source: ICodeEditor, resource: URI, selectionOrPosition?: IRange | IPosition): boolean | Promise<boolean>;
```

Comment:
Please add some doc comments explaining when this method is called (copying the comment from the register call).